source/_patterns/20-molecules/filter-bar/filter-bar.scss

```diff
@@ -0,0 +1,98 @@
+.filter-bar {
+  margin: 0;
+  height: 72px;
+
+  display: flex;
+  flex-direction: row;
+  padding: 12px 0;
+  overflow: scroll;
+
+  @include desktop-up {
+    justify-content: space-around;
+    align-content: stretch;
+    max-width: calc(100% - (2 * 32px));
+    margin: 0 32px;
+  }
+
+  @include desktop-wide {
+    max-width: 1200px;
+    margin: 0 auto;
+  }
+
+  .filter-bar__letter {
+    display: flex;
+    width: 100%;
+    margin-right: 1px;
+    border-radius: 2px;
+    padding-right: 16px;
+    padding-left: 16px;
+    color: $color-srf-warmgrey-1100;
+    white-space: nowrap;
+    align-content: center;
+    align-items: center;
+    justify-content: center;
+
+    @include desktop-up {
+      margin-right: 2px;
+      padding: 0;
+    }
+
+    &:focus,
+    &:hover {
+      background-color: $color-srf-warmgrey-50;
+    }
+
+    &:active {
+      background-color: $color-srf-warmgrey-100;
+    }
+  }
+
+  .filter-bar__letter--active {
+    background-color: $color-srf-warmgrey-100;
+    
+    &:focus,
+    &:hover {
+      background-color: $color-srf-warmgrey-100;
+    }
+  }
+
+  .filter-bar__letter--inactive {
```

Comment:
Minor nitpicking vong dem Naming her: inactive ist doch jeder Letter, der nicht active ist 🙈 Ich hätte es jetzt als `disabled` benennt.

Aber: Links zu disablen (was hier ja gemacht wird) ist eben so eine Sache: https://css-tricks.com/how-to-disable-links/

> Seriously folks, just don't do it.

Ich würde mindestens noch ein `pointer-events: none;` hinzufügen, so dass die Hover-/...-States nicht mehr greifen. Schöner wäre aber fast wenn's im Twig nicht als `<a>` sondern als `<div>` (oder whatever) gerendert wird. Ist halt mehr Aufwand :(

Reply:
Sag das mit dem Link-Disablen nicht mir, sondern jemandem anderen ;-) (siehe History des .twig-Files ;-) ). Musste es mal für die CMS-Integration in Anchors wandeln ;-)